rs-e2e/src/main/resources/assertion_definitions.json

```diff
@@ -91,6 +91,15 @@
       "rules": [
         "OBX-6 = 'µmol/L'"
       ]
+    },
+    {
+      "name": "UCSD ORU remove OBX when OBX-3.4 is 99717-5",
+      "conditions": [
+        "MSH-10 = '009'"
+      ],
+      "rules": [
+        "OBX.count() = 1"
```

Comment:
I would add an additional rule that checks that OBX-3 = "57721-3" so we know that the remaining obx is the one we expect

Reply:
Done